client/clue/clues.js

```diff
@@ -40,7 +45,10 @@ Clues.propTypes = {
   clues: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
   storyUid: React.PropTypes.string.isRequired,
 }
-export default connect(stateToProps)(Clues)
+export default connect(stateToProps, {
+  startDrag,
```

Comment:
not sure I follow, those props are there

`export default connect(stateToProps, {
  startDrag,
  dropClue
})(Clues)`